### ISSUE TITLE
Type Error when running the aptsources-cleanup.pyz

### ISSUE THREAD

**Issue by elssam** [davidfoerster/aptsources-cleanup#27]
Hello 

Please, I am getting this error when running the script:

```
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "aptsources-cleanup.pyz/__main__.py", line 12, in <module>
  File "/usr/lib/python3.5/runpy.py", line 208, in run_module
    return _run_code(code, {}, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "aptsources-cleanup.pyz/aptsources_cleanup/__main__.py", line 387, in <module>
  File "aptsources-cleanup.pyz/aptsources_cleanup/__main__.py", line 50, in main
  File "aptsources-cleanup.pyz/aptsources_cleanup/__main__.py", line 306, in handle_duplicates
  File "aptsources-cleanup.pyz/aptsources_cleanup/util/gettext.py", line 337, in __init__
    buf = fp.read()
  File "aptsources-cleanup.pyz/aptsources_cleanup/util/gettext.py", line 388, in _get_short_and_styled
    plural = v[1].split('plural=')[1]
  File "aptsources-cleanup.pyz/aptsources_cleanup/util/functools.py", line 37, in __call__
    # update_wrapper() and wraps() are tools to help write
TypeError: '_sre.SRE_Match' object is not subscriptable
```

Any help would be appriciated. 

Thanks
Samar

**Comment by elssam** [comment]
Hi, David, I forgot to say I am using Debian 9

**Comment by elssam** [comment]
I would like to install R and here are also my main issues FYI (see screenshot for errors). That's how I came to learn about your script from StackOverflow

![Screenshot from 2020-04-19 08-23-13](https://user-images.githubusercontent.com/13832075/79680370-ced98d00-8206-11ea-91cf-c6ae095dacf3.png)

**Comment by davidfoerster** [comment]
I used a feature introduced in Python v3.6 by accident. Sorry for that. Fixed in e0cc7deb1152a7e126ef50a4d06fbab261992021 (included in v0.1.7.1).

On a slightly different note, I don’t understand why you want to de-duplicate Apt source lines since there are no such warnings from Apt. Instead, it complains about a package dependency problem that you need to address (and which is out of scope for this issue report or the purpose of *aptsources-cleanup*).
